src/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>LeaderBoard</title>
+  </head>
+
+  <body>
+    <header class="tittle">
+      <h1>Leaderboard</h1>
+    </header>
+    <div class="main-container">
+      <section class="recent-score-section">
+        <div class="recent-score">
+          <h2>Recent score</h2>
+          <button type="button" id="refresh">Refresh</button>
+        </div>
+        <div class="list-score">
+          <ul>
+            <li> name: 100</li>
+            <li> name: 20</li>
```

Comment:
Please find below the error received when I check your app in the browser:

![image](https://user-images.githubusercontent.com/10332499/167712639-30d2b5ff-14c2-4987-aea4-ae3ae541a83c.png)

Kindly fix the error.

Reply:
@micheaol ,
 kindly advise me on what to do cause on my end it works properly when npm start and I have installed all the necessary loader for HTML

Reply:
@mandelatuks,

I think this docs might help : https://stackoverflow.com/questions/43494794/webpack-html-webpack-plugin-error-child-compilation-failed

Reply:
@micheaol 
worked on it didn't have HTML loader thank you